pkg/deployments/tasks/20230315-silo-linear-pool/index.ts

```diff
@@ -26,16 +25,23 @@ export default async (task: Task, { force, from }: TaskRunOptions = {}): Promise
     // Pools are automatically verified. We however don't run any of this code in CHECK mode, since we don't care about
     // the contracts deployed here. The action IDs will be checked to be correct via a different mechanism.
 
-    // TODO: Fix Mocks
+    // A Silo require a Silo Repository
+    const mockSiloRepoArgs = [0, 0];
+    const mockSiloRepo = await task.deployAndVerify('MockSiloRepository', mockSiloRepoArgs, from, force);
+
+    // shareTokens require a Silo liquidity pool
+    const mockSiloArgs = [mockSiloRepo, input.WETH];
+    const mockSilo = await task.deployAndVerify('MockSilo', mockSiloArgs, from, force);
+
     // SiloLinearPools require an Silo Token
-    const mockShareTokenArgs = ['DO NOT USE - Mock Share Token', 'TEST', 18, input.WETH];
```

Comment:
You can keep the "DO NOT USE". In this part, we are deploying a mock pool and token to generate the actionIds of the pool, so we don't want anyone to use these mocked tokens and pools